### PR TITLE
feat: add About, Events, Footer section & fix: minor typography issues

### PR DESCRIPTION
**Pull request by ORKO06**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Added About Component in shared as it can be reused.
Added Event Category Section and Footer Component
Fixed minor typography issues and import error.

❤️Thank you!

### Post merge checklist
- [x] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by abhibhaw** [comment]
Footer needs a bit more space at the bottom. For events sec don't scroll full thing, keep title fixed and scroll the cards

File: src/pages/playground.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import React, { useState } from 'react';
 import { FaqSection, FlexSection, HeroSection } from '../components';
+import Footer from '../components/marginals/Footer/Footer';
```
Comment (abhibhaw) [inline src/pages/playground.js]: Line 2 @ratnakar5938 is importing everything from the component module, you're importing relative to the absolute path. Please fix it to one format.

**Review by abhibhaw — CHANGES_REQUESTED** [review]

**Comment by riteshsp2000** [comment]
small changes here and there...
- make the entire card clickable
- make the entire faq clickable
- add some border radius to the about section video
- on mobile page add some gap between the sections in footer

**Review by riteshsp2000 — APPROVED** [review]

**Review by abhibhaw — APPROVED** [review]
LGTM

**Pull request merged**